### PR TITLE
Feature/neo4j container

### PR DESCRIPTION
**Pull request by alesr**
Implement scripts to manage a Neo4j instance running on a Docker container.

In the `config.yml` the password must be replaced to `neo4jdev`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-# Scout [![Build Status](https://travis-ci.org/project-regista/scout.svg?branch=master)](https://travis-ci.org/project-regista/scout)
\ No newline at end of file
+# Scout [![Build Status](https://travis-ci.org/project-regista/scout.svg?branch=master)](https://travis-ci.org/project-regista/scout)
+
+
+## Howto:
+
+Start a frest new instance of Neo4j running in a Docker container
```
Comment (coppetti) [inline README.md]: fresh

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.DEFAULT_GOAL := help
+
+PROJECT_NAME := regista
+IMAGE_NAME := scout
+
+NEO4J_CONTAINER_NAME := neo4j-"${PROJECT_NAME}"
+NEO4J_HTTP_PORT := 7474
```
Comment (coppetti) [inline Makefile:7]: Maybe 7475, and 7688 (or something like this)
in the case we have a local instance for neo4j running

Comment (alesr) [reply Makefile:7]: I get this error when try a different port mapping:

```
 WebSocket connection failure. Due to security constraints in your web browser, the reason for the failure is not available to this Neo4j Driver. Please use your browsers development console to determine the root cause of the failure. Common reasons include the database being unavailable, using the wrong connection URL or temporary network problems. If you have enabled encryption, ensure your browser is configured to trust the certificate Neo4j is configured to use. WebSocket `readyState` is: 3
```

I will leave it using the defaults ports for now.

Comment (coppetti) [reply Makefile:7]: ok, so lets merge :D

**Review by coppetti — APPROVED** [review]

**Pull request merged**